api/apps/api/test/upload-feature/upload-feature.fixtures.ts

```diff
@@ -264,6 +268,70 @@ export const getFixtures = async () => {
         .send();
     },
     // ASSERT
+    ThenWaitForApiEvent: (topic: string, kind: API_EVENT_KINDS) => {
```

Comment:
just a tiny naming note: for this and for all the following fixture functions that wait for specific API events to be fired, I think they are used as something inbetween acting and asserting, basically holding off until we can do the core assertions... (you've placed the functions within the assert section here, but often within the act section within tests).
No need to do anything, but my suggestion would be to treat them as assertions throughout, with names such as `ThenA<kind>EventIsSubmitted`, but well, yes, it's hair-splitting on naming, so up to you 😄